fxmanifest.lua

```diff
@@ -7,6 +7,7 @@ version '1.0.0'
 shared_scripts {
 	'@qb-core/shared/locale.lua',
 	'locales/en.lua',
+    'locales/*.lua',
```

Comment:
Please fix the indentations. Looks like tab was used previously. Would be great if you could replace all tabs with 4 spaces. Thanks